src/rgw/rgw_rados.cc

```diff
@@ -3822,7 +3822,7 @@ int RGWRados::init_rados()
     }
   }
 
-  auto handles = std::vector<librados::Rados>{cct->_conf->rgw_num_rados_handles};
+  auto handles = std::vector<librados::Rados>{static_cast<unsigned int>(cct->_conf->rgw_num_rados_handles)};
```

Comment:
s/unsigned int/std::size_t/

Reply:
I've made this change, thanks for the feedback.